paasta_tools/deploy_marathon_services

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 if am_i_mesos_leader >/dev/null; then
-  list_marathon_service_instances | shuf | xargs -n 1 -r -P 5 setup_marathon_job
+  list_marathon_service_instances | xargs -n 64 -r -P 4 setup_marathon_job
```

Comment:
Can you still leave in the shuf?
The shuf protects us in a different way, so that if we do have a service that gets stuck and hurts us, at least on the next run there is a chance for randomness to help give others a chance.